service/controller.go

```diff
@@ -2132,9 +2139,75 @@ func (s *service) GetCapacity(
 			"Unable to get capacity: %s", err.Error())
 	}
 
+	systemID := ""
+	for key, value := range params {
+		if strings.EqualFold(key, KeySystemID) {
+			systemID = value
+			break
+		}
+	}
+
+	maxVolSize, err := s.getMaximumVolumeSize(systemID)
+	if err != nil {
+		Log.Debug("GetMaxVolumeSize returning error ", err)
+	}
+	maxVolSizeinbps := maxVolSize * bytesInGiB
```

Comment:
@KshitijaKakde Perform this (maxVolSizeinbps := maxVolSize * bytesInGiB) operation after if maxVolSize < 0  check.

Reply:
Done